ChangeLog

```diff
@@ -1,3 +1,10 @@
+Sun Oct 03 01:50:00 -03:00 2021  IFo Hancroft <contact@ifohancroft.com>
```

Comment:
We do not change this file until the release of a new version, which has not been discussed yet and the one in charge of this is @eribertomota 
However, the version number that you changed indicates a patch, which is not.

Reply:
I have not changed the existing functionality. I have added an extra option.

As for the version number - what is the appropriate version number?

Reply:
See: https://semver.org/

Reply:
Thank you!

Should I remove what I added from ChangeLog or should I change it to 1.7 (since that's the correct way according to semver for new features with backwards compatiblity)

Reply:
We never modify the files `ChangeLog` , `create-man.sh` and `scrot.1`, are files that are modified when launching a new release, that is in charge of @eribertomota 

You can modify `scrot.txt`

Reply:
Ah! Got it. Thank you!
I will revert the changes when I wake up later today.

Reply:
I think it should be ok now.